include/seqan3/argument_parser/detail/format_parse.hpp

```diff
@@ -222,11 +222,30 @@ class format_parse : public format_base
         return (std::find_if(begin_it, end_of_options_it,
```

Comment:
A few lines above in the documentation:

```
     * Note: The `id` is compared to the prefix of each value in the list, such
     *       that "-idValue" arguments are correctly identified.	     *       that "-idValue" arguments are correctly identified.
```

`"-idValue"` won't work anymore.
Also, maybe you can use `\note`? Or at least add an explicit `\details`

Reply:
I need to adjust the documentation! Thanks for spotting this.

in you example there is one twirk. we do not allow multi character short ids so "-idValue" will never occur.
Valid short id value pairs: `-iValue`, `-i=Value`, `-i Value`
Valid long id value pairs (now) `--id=Value`, `--id Value`

Reply:
ops that was my example not yours... anyway I'll adjust it